docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,254 @@
+It is simple enough that you can easily integrate its use into your own automation (Terraform, Chef, Puppet, etc).
+
+**The `kubeadm` tool is currently in alpha but please try it out and give us [feedback](/docs/getting-started-guides/kubeadm/#feedback)!**
+
+## Prerequisites
+
+1. One or more machines running Ubuntu 16.04 or CentOS 7
+1. 1GB or more of RAM per machine (any less will leave little room for your apps)
+1. Full network connectivity between all machines in the cluster (public or private network is fine)
+
+## Objectives
+
+* Install a secure Kubernetes cluster on your machines
+* Install a pod network on the cluster so that application components (pods) can talk to each other
+* Install a sample microservices application (a socks shop) on the cluster
+
+## Instructions
+
+### (1/4) Installing kubelet and kubeadm on your hosts
+
+You will install the following packages on all the machines:
+
+* `docker`: the container runtime, which Kubernetes depends on.
+* `kubelet`: the most core component of Kubernetes.
+  It runs on all of the machines in your cluster and does things like starting pods and containers.
+* `kubectl`: the command to control the cluster once it's running.
+  You will only use this on the master.
+* `kubeadm`: the command to bootstrap the cluster.
+
+For each host in turn:
+
+<!--
+    # curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+    # cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
+    deb http://packages.cloud.google.com/apt kubernetes-xenial main
+    EOF
+    # apt-get update
+    # apt-get install -y kubeadm docker.io§
+-->
+
+
+* SSH into the machine and become `root` if you are not already (for example, run `sudo su -`).
+* If the machine is running Ubuntu 16.04, run:
+
+      # apt-get install -y docker.io socat apt-transport-https
+      # curl -s -L \
+        https://storage.googleapis.com/kubeadm/kubernetes-xenial-preview-bundle.txz | tar xJv
+      # dpkg -i kubernetes-xenial-preview-bundle/*.deb
+
+   If the machine is running CentOS 7, run:
+
+      # cat <<EOF > /etc/yum.repos.d/k8s.repo
+      [kubelet]
+      name=kubelet
+      baseurl=http://files.rm-rf.ca/rpms/kubelet/
+      enabled=1
+      gpgcheck=0
+      EOF
+      # yum install docker kubelet kubeadm kubectl kubernetes-cni
+      # systemctl enable docker && systemctl start docker
+      # systemctl enable kubelet && systemctl start kubelet
+
+The kubelet is now restarting every few seconds, as it waits in a crashloop for `kubeadm` to tell it what to do.
+
+### (2/4) Initializing your master
+
+The master is the machine where the "control plane" components run, including `etcd` (the cluster database) and the API server (which the `kubectl` CLI communicates with).
+All of these components run in pods started by `kubelet`.
+
+To initialize the master, pick one of the machines you previously installed `kubelet` and `kubeadm` on, and run:
+
+     # kubeadm init --use-kubernetes-version v1.4.0-beta.11
```

Comment:
Need to remove `--use-kubernetes-version v1.4.0-beta.11` when @mikedanese's official packages land.